qiskit/circuit/instructionset.py

```diff
@@ -186,6 +186,26 @@ def c_if(self, classical: Union[Clbit, ClassicalRegister, int], val: int) -> "In
         Raises:
             CircuitError: if the passed classical resource is invalid, or otherwise not resolvable
                 to a concrete resource that these instructions are permitted to access.
+
+        Example:
+        .. jupyter-execute::
```

Comment:
Does this format as expected? I thought the napolean plugin has trouble with things like this. But I could be misremembering.